deviceModeInit.js

```diff
@@ -376,10 +390,10 @@ var rudderTracking = (function () {
       })
       .then(() => {
         updateTimeStampForIdentifierEvent();
-        console.log("Successfully sent identifier event to rudderstack");
+        console.debug("Successfully sent identifier event to rudderstack");
       })
       .catch(() => {
-        console.log("Failed to sent identifier event to rudderstack");
+        console.error("Failed to sent identifier event to rudderstack");
```

Comment:
These are our internal calls right? should we show errors in the console for them?

Reply:
Would debug suffice it ?